### PR TITLE
Add read more/show less button & posibility to hide all columns

### PR DESCRIPTION
**Pull request by daianamezdrea**
Read more/show less is still WIP, I need to add a length condition, but it's good to merge. And the commented parts are for experimental radio buttons (to see how it behaves).

### CODE REVIEW

File: tools.html.erb [hunk 1]
```diff
@@ -39,21 +38,42 @@
...
+  border: 1px solid #ddd;
+}
 
 }
 .word_break {
   word-break: break-all;
   width: auto;
 }
-.button {
+/*.button {
```
Comment (carabasdaniel) [inline tools.html.erb:76]: Please remove unused block

File: tools.html.erb [hunk 2]
```diff
@@ -88,47 +108,172 @@
...
+    height: auto;
+}
+.showContent{
+    height: auto;
+}
+.show-more {
+    padding: 10px 0;
+    text-align: center;
+}
+ /*hereeeeeeeeeeee */
```
Comment (carabasdaniel) [inline tools.html.erb:165]: Remove this comment

File: tools.html.erb [hunk 3]
```diff
@@ -39,21 +38,42 @@
...
+         linkText = "Show more";
+         $content.removeClass('showContent');
+         $content.addClass('hideContent');
+      };
+
+      $this.text(linkText);
+   });
+
 </script>
 <style>
```
Comment (carabasdaniel) [inline tools.html.erb:61]: You might want to use a stylesheet here

File: tools.html.erb [hunk 4]
```diff
@@ -39,21 +38,42 @@
       e.preventDefault();
     })
 } );
+
+$(document).on("click", '.show-more a', function(e) {
```
Comment (carabasdaniel) [inline tools.html.erb:42]: As this script is getting bigger you might want to consider moving this to a js file

**Review by carabasdaniel — CHANGES_REQUESTED** [review]

File: tools.html.erb (hunk 4 above)
Comment (daianamezdrea) [reply tools.html.erb:42]: Yes, totally, I want to have everything separated in html/css/js files.

**Pull request merged**